src/ProcessedStory.ts

```diff
@@ -75,6 +80,14 @@ export default class ProcessedStory {
     return this.parameters.runOptions;
   }
 
+  /**
+   * Context passed to `axe.run`.
+   * @see https://www.deque.com/axe/core-documentation/api-documentation/#context-parameter
+   */
+  get context() {
+    return this.parameters.context;
```

Comment:
Not sure if this is checked elsewhere, but is `.parameters` guaranteed to exist here? (assuming typescript would be complaining if not)

Reply:
Yeah, it's initialized in this object's constructor